src/consent.ts

```diff
@@ -63,4 +63,35 @@ export class Consent {
     );
     document.head.prepend(script);
   }
+
+  async enableGoogleTagManager(
+    id: string,
+    role: ConsentCategory = "statistic"
+  ): Promise<void> {
+    await this.consentFor(role);
+    window.gtag({
+      "gtm.start": new Date(),
+      event: "gtm.js",
+    });
+    window.gtag("config", id, { send_page_view: false }); // page view is disabled because it's tracked via the analytics stimulus controller already
+    const script = document.createElement("script");
+    script.async = true;
+    script.setAttribute(
+      "src",
+      `https://www.googletagmanager.com/gtm.js?id=${id}`
+    );
+    document.head.prepend(script);
+
```

Comment:
Do we have to add additional code so we can track Turbo navigation?

Reply:
I'm not sure. Jens didn't bring it up in the pairing session yesterday.

Reply:
What's the reasoning behind that?

Reply:
@JensRavens Do you have any input on this?

Reply:
yes, it's already handled by the stimulus controller that was implemented for google analytics and can be reused without any modifications